### ISSUE TITLE
Errors in test build restoring TestWrappers\baseservices.compilerservices\baseservices.compilerservices.XUnitWrapper.csproj

### ISSUE THREAD

**Issue by MattGal** [dotnet/coreclr#19142]
@dotnet-mc-bot commented on [Thu Jul 26 2018](https://github.com/dotnet/core-eng/issues/3944)

Opened on behalf of @MattGal

```
2018-07-26T13:21:23.9717211Z   Generating MSBuild file E:\A\_work\3\s\/bin/tests/obj/Windows_NT.x64.Release/Managed//baseservices.compilerservices.XUnitWrapper\baseservices.compilerservices.XUnitWrapper.csproj.nuget.g.props.
2018-07-26T13:21:23.9809863Z   Generating MSBuild file E:\A\_work\3\s\/bin/tests/obj/Windows_NT.x64.Release/Managed//baseservices.compilerservices.XUnitWrapper\baseservices.compilerservices.XUnitWrapper.csproj.nuget.g.targets.
2018-07-26T13:21:24.0464838Z   Restore completed in 32.17 sec for E:\A\_work\3\s\bin\tests\Windows_NT.x64.Release\TestWrappers\baseservices.compilerservices\baseservices.compilerservices.XUnitWrapper.csproj.
2018-07-26T13:21:24.1293187Z E:\A\_work\3\s\bin\tests\Windows_NT.x64.Release\TestWrappers\baseservices.compilerservices\baseservices.compilerservices.XUnitWrapper.csproj : warning NU1603: baseservices.compilerservices.XUnitWrapper depends on Microsoft.NETCore.App (>= 3.0.0-preview1-26720-05) but Microsoft.NETCore.App 3.0.0-preview1-26720-05 was not found. An approximate best match of Microsoft.NETCore.App 3.0.0-preview1-26721-01 was resolved.
2018-07-26T13:21:24.1294561Z E:\A\_work\3\s\bin\tests\Windows_NT.x64.Release\TestWrappers\baseservices.compilerservices\baseservices.compilerservices.XUnitWrapper.csproj : error : The project was restored using Microsoft.NETCore.App version 3.0.0-preview1-26721-01, but with current settings, version 3.0.0-preview1-26720-05 would be used instead.  To resolve this issue, make sure the same settings are used for restore and for subsequent operations such as build or publish.  Typically this issue can occur if the RuntimeIdentifier property is set during build or publish but not during restore.
2
```

Warnings: 1
Status Message: failed

Build : 3.0 - 20180726.04 (Product Build)
Failing configurations:
- Alpine3.6
  - Build-Tests-R2R-Release
  - Build-Tests-Release
- RedHat6
  - Build-Tests-R2R-Release
  - Build-Tests-Release
- RedHat 7
  - Build-Tests-R2R-Release
  - Build-Tests-Release
- OSX
  - Build-Tests-R2R-Release
  - Build-Tests-Release
- Windows
  - Build-Tests-R2R-Release-arm64
  - Build-Tests-Release-arm64
  - Build-Tests-R2R-Release
  - Build-Tests-Release
[Mission Control Build Info](https://mc.dot.net/#/product/netcore/30/source/official~2Fcoreclr~2Fmaster~2F/type/build~2Fproduct~2F/build/20180726.04/workItem/Orchestration/analysis/external/Link)

---

@MattGal commented on [Thu Jul 26 2018](https://github.com/dotnet/core-eng/issues/3944#issuecomment-408192726)

Oops meant to move this to CoreCLR.

**Comment by MattGal** [comment]
@RussKeldorph  FYI; this is breaking lots of test builds presently

**Comment by 4creators** [comment]
This is dupe of bug which was fixed in #19130 . 

@MattGal @RussKeldorph

**Comment by RussKeldorph** [comment]
Fixed by #19130
